node/src/actors/chain_manager/mining.rs

```diff
@@ -72,18 +72,14 @@ impl ChainManager {
             _ => return,
         };
 
-        if beacon.checkpoint > current_epoch {
+        if beacon.checkpoint >= current_epoch {
             // We got a block from the future
             error!(
                 "The current highest checkpoint beacon is from the future ({:?} > {:?})",
```

Comment:
```suggestion
                "The current highest checkpoint beacon is from the future ({:?} >= {:?})",
```

And add a comment explaining why >= is the same as >